src/amo/reducers/userAbuseReports.js

```diff
@@ -0,0 +1,175 @@
+/* @flow */
+import type { UserType } from 'amo/reducers/users';
+
+export const HIDE_USER_ABUSE_REPORT_UI = 'HIDE_USER_ABUSE_REPORT_UI';
+export const LOAD_USER_ABUSE_REPORT = 'LOAD_USER_ABUSE_REPORT';
+export const SEND_USER_ABUSE_REPORT = 'SEND_USER_ABUSE_REPORT';
+export const SHOW_USER_ABUSE_REPORT_UI = 'SHOW_USER_ABUSE_REPORT_UI';
+
+
+type HideUserAbuseReportUIType = {|
+  user: UserType,
+|};
+
+export function hideUserAbuseReportUI(
+  { user }: HideUserAbuseReportUIType = {}
```

Comment:
It looks to me like only `userId` is required for this action

Reply:
Indeed, but I thought passing around the user object consistently made for a nicer API and made the types declared in flow easier to manage (everything is just a `UserType`). I didn't see the harm in passing a user to the action and having the reducer handle getting what it needs.